Sources/Linq2DynamoDb.DataContext.Tests/EntityManagementTests/EntityCreationTests.cs

```diff
@@ -33,17 +33,18 @@ public void DataContext_EntityCreation_PersistsRecordToDynamoDb()
             Assert.IsNotNull(storedBook);
         }
 
-        [Ignore("This behavior is currently expected. SubmitChanges() uses DocumentBatchWrite, which only supports PUT operations, which by default replaces existing entities")]
         [Test]
-        [ExpectedException(typeof(InvalidOperationException), ExpectedMessage = "cannot be added, because entity with that key already exists", MatchType = MessageMatch.Contains)]
+        [ExpectedException(typeof(AggregateException))]
```

Comment:
hmmm i'd probably want to catch+flatten any aggregate exceptions that get thrown internally.
unless a method returns a `Task` (is `Async`), a user should not expect that method to throw an `AggregateException`.